src/main.ts

```diff
@@ -63,9 +63,18 @@ export default class CopyUrlInPreview extends Plugin {
           document,
           "contextmenu",
           "img",
-          this.onClickImage.bind(this)
+          this.onClickImage_contextmenu.bind(this)
         )
-      )
+      );
+
+      this.register(
+        onElement(
+          document,
+          "mouseup",
+          "img",
+          this.onClickImage_click.bind(this)
```

Comment:
a better name would be "onClickImage_mouseup". It's named _click though because I forgot to rename when I replaced the event type from "click" to "mouseup", because "click" didn't work for middle mouse